src/envs/pybullet_cover.py

```diff
@@ -228,7 +228,7 @@ def step(self, action: Action) -> State:
         # constraint before we can call PyBullet.
         # Use self._pybullet_robot2 to run forward kinematics, since that
         # method shouldn't be run on the client that is doing simulation.
-        _, ry, rz = self._pybullet_robot2.forward_kinematics(action.arr)
+        _, ry, rz = self._pybullet_robot2.forward_kinematics(list(action.arr))
```

Comment:
should we use `tolist()` instead? from https://numpy.org/doc/stable/reference/generated/numpy.ndarray.tolist.html:
"For a 1D array, a.tolist() is almost the same as list(a), except that tolist changes numpy scalars to Python scalars"